docs/readmes/orc8r/deploy_using_juju.md

```diff
@@ -57,6 +68,15 @@ applications:
 
 Deploy Orchestrator:
 
+<!-- markdownlint-disable MD028 -->
+
+> **Note**: The magma-orc8r bundle is available on edge and beta channels.
```

Comment:
I'd probably remove this note as it'll soon be obsolete

Reply:
Done.